backend/tests/data_layer/database/crud/sqlite/test_smartapi_crud.py

```
@@ -48,21 +48,38 @@ def test_get_smartapi_tokens_by_any_and_all_condition():
             lot_size=1,
         )
 
+        # Test get_smartapi_tokens_by_any_condition with symbol
         result = get_smartapi_tokens_by_any_condition(symbol="INFY")
         for item in result:
             assert "INFY" == item.symbol
 
+        # Test get_smartapi_tokens_by_any_condition with exchange
         result = get_smartapi_tokens_by_any_condition(exchange="BSE")
         for item in result:
             assert "BSE" == item.exchange
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `item.exchange == "BSE"`

(SIM300)